plasmapy/particles/ionization_state.py

```diff
@@ -166,7 +164,7 @@ class IonizationState:
         an element, isotope, or ion; or an integer representing the
         atomic number of an element.
 
-    ionic_fractions: `~numpy.ndarray`, `list`, `tuple`, or `~astropy.units.Quantity`; optional
+    ionic_levels: `~numpy.ndarray`, `list`, `tuple`, or `~astropy.units.Quantity`; optional
```

Comment:
```suggestion
    ionic_fractions: `~numpy.ndarray`, `list`, `tuple`, or `~astropy.units.Quantity`; optional
```
This also should be changed back to `ionic_fractions`.

Reply:
The argument, but not the field of `IonizationState`, if I understand correctly?

Reply:
Oh, okay, I get it now. Sorry, tired.